androidlibrary_lib/src/main/res/color/btn_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/colorBtnDisabled" android:state_enabled="false" />
+    <item android:color="@color/colorPrimaryBackground" android:state_enabled="true" />
+</selector>
```

Comment:
Small issue, file does not have a blank line at the end and GitHub is pointing it out. Similar issue with other xml files

Reply:
Resolved